java/app/grapheneos/setupwizard/view/activity/WelcomeActivity.kt

```diff
@@ -54,7 +55,11 @@ class WelcomeActivity : SetupWizardActivity(R.layout.activity_welcome) {
     override fun setupActions() {
         language.setOnClickListener { WelcomeActions.showLanguagePicker(this) }
         accessibility.setOnClickListener { WelcomeActions.accessibilitySettings(this) }
-        emergency.setOnClickListener { WelcomeActions.emergencyCall(this) }
+        if (packageManager.hasSystemFeature(PackageManager.FEATURE_TELEPHONY_CALLING)) {
+            emergency.setOnClickListener { WelcomeActions.emergencyCall(this) }
+        } else {
+            emergency.visibility = View.GONE
```

Comment:
this change is against the design pattern in use.

`setupActions()` is only supposed to setup event handlers. hiding or showing a view is here is not a good code.
do inside `bindViews(..)` pls or even better, define a `setupViews(..)` in base class and override here.